lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -407,7 +409,7 @@ def add_app_targets
               spec_name = app_spec.parent.name
               subspec_name = target.subspec_label(app_spec)
               app_target_label = target.app_target_label(app_spec)
-              platform = target.platform
+              platform = Platform.new(target.platform.symbolic_name, target.deployment_target_for_non_library_spec(app_spec))
```

Comment:
maybe add a `platform_for_spec` method or somethign like that?

Reply:
will do.